azurelinuxagent/common/utils/fileutil.py

```diff
@@ -52,6 +52,13 @@ def copy_file(from_path, to_path=None, to_dir=None):
     return to_path
 
 
+def move_file(from_path, to_path=None, to_dir=None):
+    if to_path is None:
+        to_path = os.path.join(to_dir, os.path.basename(from_path))
```

Comment:
I think `shutil.move` already does this if the destination is a directory.

I think it looks weird to have both `to_path` and `to_dir` as parameters since `to_path` "wins" if you specify both.

Reply:
You're right.  I deleted copy_file and move_file, and just used shutil.